features/api/3ds_authorization.feature

```diff
@@ -1,56 +1,57 @@
+            | address1         | 4                                                                                                                                                                                                |
+            | address2         | Braford Gardens                                                                                                                                                                                  |
+            | address3         | Shenley Brook End                                                                                                                                                                                |
+            | postalCode       | MK137QJ                                                                                                                                                                                          |
+            | city             | Milton Keynes                                                                                                                                                                                    |
+            | state            | Buckingamshire                                                                                                                                                                                   |
+            | countryCode      | GB                                                                                                                                                                                               |
+            | shopperIPAddress | 123.123.123.123                                                                                                                                                                                  |
+            | sessionId        | 0215ui8ib1                                                                                                                                                                                       |
+            | email            | lpanainte+test@inviqa.com                                                                                                                                                                        |
+            | acceptHeader     | text/html                                                                                                                                                                                        |
+            | userAgentHeader  | Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko)                                                                                                                           |
+            | dynamic3DS       | do3DS                                                                                                                                                                                            |
```

Comment:
Is it worth having this as a boolean flag (true/false) instead of library integrators requiring knowledge about the actual text value sent via the API?